### PR TITLE
AML-3763 About Your MoU

### PR DESCRIPTION
**Pull request by Howesy77**

### CODE REVIEW

**Comment by Howesy77** [comment]
Content is confirmed as ok

File: src/SFA.DAS.EmployerAccounts.Web/Helpers/ControllerConstants.cs [hunk 1]
```diff
@@ -2,7 +2,9 @@
 {
     public static class ControllerConstants
     {
-        public const string AboutYourAgreement = "AboutYourAgreement";
+        public const string AboutYourAgreementActionName = "AboutYourAgreement";
```
Comment (dashton82) [inline src/SFA.DAS.EmployerAccounts.Web/Helpers/ControllerConstants.cs]: its probably a bigger refactoring exercise, but splitting these into two classes for view names and actions, and not calling the property names xxxViewName, you could possibly do:
```
public static string AboutYourAgreement => nameof(AboutYourAgreement);
```

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/AboutYourMou.cshtml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+﻿@using SFA.DAS.Authorization
+@using SFA.DAS.Authorization.Mvc
+@using SFA.DAS.EmployerAccounts.Web.Helpers
+
+@model OrchestratorResponse<EmployerAgreementViewModel>
+
+@{
+    ViewBag.PageID = "mou-about";
+    ViewBag.Title = "About your MoU";
```
Comment (dashton82) [inline src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/AboutYourMou.cshtml]: is `MoU` correct to be shown in the title?

File: src/SFA.DAS.EmployerAccounts.Web/Helpers/ControllerConstants.cs (hunk 1 above)
Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerAccounts.Web/Helpers/ControllerConstants.cs]: I agree, it's got to the size where we should split it out but not for now.  I'll do it the next time I'm in that class.

File: src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/AboutYourMou.cshtml (hunk 2 above)
Comment (MichaelYoung1981) [reply src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/AboutYourMou.cshtml]: I'm pretty sure it is...EoI, MoU...don't you love interchangeable terms.

Comment (Howesy77) [reply src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/AboutYourMou.cshtml]: It should probably be document - didn't have a design when this page was done :)

**Pull request closed without merging**